kernel/patches/5.4.x/0003-arm64-mm-Enable-memory-hot-remove.patch

```diff
@@ -0,0 +1,498 @@
+From ba91422b18892bceacf3b4aa60354cf36fcabf9b Mon Sep 17 00:00:00 2001
```

Comment:
I'm sure this has probably been discussed - but, do we do hot remove on Kata - I thought we pretty much only ever added, as remove can be very expensive (as you need to 'clear out' any used pages in the 'dimm' you want to remove).
/cc @jcvenegas for any thoughts.

Reply:
Hi~ @grahamwhaley 
The reason that I enabled memory hot remove is that `ZONE_DEVICE` is dependent on it, and 
NVDIMM/DAX needs `ZONE_DEVICE`. ;). maybe I should elaborate more in commit message.